compiler/src/org.graalvm.compiler.lir.aarch64/src/org/graalvm/compiler/lir/aarch64/AArch64AtomicMove.java

```diff
@@ -133,25 +154,26 @@ public AtomicReadAndAddOp(AArch64Kind kind, AllocatableValue result, Allocatable
         @Override
         public void emitCode(CompilationResultBuilder crb, AArch64MacroAssembler masm) {
             assert accessKind.isInteger();
-            final int size = accessKind.getSizeInBytes() * Byte.SIZE;
+            final int srcSize = accessKind.getSizeInBytes() * Byte.SIZE;
```

Comment:
Instead of srcSize and dstSize, can you use more descriptive names, like memAccessSize and addSize?